supervisord/tests/common.py

```diff
@@ -2,15 +2,23 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 
-from datadog_checks.checks.base import AgentCheck
-from datadog_checks.supervisord.supervisord import SupervisordCheck
+import os
 
+from datadog_checks.checks.base import ServiceCheck
```

Comment:
@ofek this is confusing for me now. We created `ServiceCheck` constants (i am still not sure why https://github.com/DataDog/integrations-core/pull/2960#issuecomment-454892015) and now we have two ways to reference the same thing. What is the reasoning behind it? Could we make it so we just have one way?